src/pxl_scripts/bpftrace/tcp_drops/data.pxl

```diff
@@ -20,7 +20,7 @@ import pxtrace
 import px
 
 # Adapted from https://github.com/iovisor/bpftrace/blob/master/tools/tcpdrop.bt
```

Comment:
Could we add a comment here (and in the other file) mentioning how kernel backporting can cause potential compatibility issues?